### PR TITLE
Code ready to Production

### PR DESCRIPTION
**Pull request by tdmitchell**

### CODE REVIEW

File: README.txt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+-JS and CSS files for each component are stored inside each Component Folder in the src folder
+-
+
+
+5. APIs
+5.1 API Ninjas - City API: https://api-ninjas.com/api/city
+
+- URL: https://api.api-ninjas.com/v1
+
+- Endpoits:
```
Comment (mwazir) [inline README.txt]: Typo here, should be "Endpoints".

File: README.txt [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        * Parameters: client_id: "your key",name: 'country name'
+    
+
+- Authentication: 
+
+5.2 Unsplash API: https://unsplash.com/documentation#like-a-photo
+
+- URL: https://api.unsplash.com/
+
+- Endpoits:
```
Comment (mwazir) [inline README.txt]: Typo here, should be "Endpoints".

File: src/App.js [hunk 3]
```diff
@@ -1,23 +1,132 @@
...
+
+    if (city) {
+      axios({
+        url: `https://api.api-ninjas.com/v1/city?name=${city}`,
+        method: "GET",
+        headers: {
+          "X-Api-Key": "OMPD0sHj09N5XJV/prIM3Q==zKLddhyIL406cdHa",
+        },
+      }).then((res) => {
+        // console.log("Res", res);
```
Comment (mwazir) [inline src/App.js]: Can remove console.log

File: src/App.js [hunk 4]
```diff
@@ -1,23 +1,132 @@
...
+        const responseData = res.data[0];
+
+        console.log("responseData", responseData);
```
Comment (mwazir) [inline src/App.js]: Can remove console.log

File: src/App.js [hunk 5]
```diff
@@ -1,23 +1,132 @@
...
+
+        setSelectedInfo(responseData);
+        // setCity("Sao Paulo"); //           //It changes city state and gets new data from the API
```
Comment (mwazir) [inline src/App.js]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/App.js [hunk 6]
```diff
@@ -1,23 +1,132 @@
...
+        // console.log(withOrientation);
+        // console.log(allPhotos); //RETURNS AN EMPTY ARRAY, WHY ????????
+      });
+    }
+  }, [city]);
+
+  // Get info from Form
+  const getInfo = (e, cityName) => {
+    e.preventDefault();
+    // console.log("city From getInfo", cityName);
```
Comment (mwazir) [inline src/App.js]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/App.js [hunk 7]
```diff
@@ -1,23 +1,132 @@
...
+    setCity(cityName);
+    // let newCityInfo = selectedInfo;
+    // getPhotos(e, cityName);
+    // return newCityInfo;
+
+    // setSelectedInfo(apiCityInformation);
+    // console.log("apiCityInformation", apiCityInformation);
```
Comment (mwazir) [inline src/App.js]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/App.js [hunk 8]
```diff
@@ -1,23 +1,132 @@
...
+        },
+      }).then((res) => {
+        const responseData = res.data.results;
+
+        setCityPhotos(responseData);
+
+        console.log("res", res);
+        console.log("res.data.results", res.data.results);
+        // console.log(withOrientation);
+        // console.log(allPhotos); //RETURNS AN EMPTY ARRAY, WHY ????????
```
Comment (mwazir) [inline src/App.js]: Not sure if this is still being used, but if not then the commented code can be removed and console.log can be removed

File: src/App.js [hunk 9]
```diff
@@ -1,23 +1,132 @@
...
+  //States
+  // const [city, setCity] = useState("Rio de Janeiro");
+  const [city, setCity] = useState("");
+  const [selectedInfo, setSelectedInfo] = useState({
+    // name: "Toront",
+    // country: "Canada1",
+    // population: 200000,
+    // latitude: 51.05,
+    // longitude: -114.067,
+    // is_capital: false,
```
Comment (mwazir) [inline src/App.js]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/App.js [hunk 10]
```diff
@@ -1,23 +1,132 @@
...
+  //   name: "Toronto",
+  //   country: "Canada",
+  //   population: 200000,
+  //   latitude: 51.05,
+  //   longitude: -114.067,
+  //   is_capital: false,
+  // };
+
+  //States
+  // const [city, setCity] = useState("Rio de Janeiro");
```
Comment (mwazir) [inline src/App.js]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/App.js [hunk 11]
```diff
@@ -1,23 +1,132 @@
...
+    // console.log("apiCityInformation", apiCityInformation);
+  };
+
+  // const getPhotos = (e, cityName) => {
+  //   // e.preventDefault();
+  //   console.log("city From getPhoto", cityName);
+  //   setCity(cityName);
+  //   let selectedPhotos = cityPhotos;
+  //   return selectedPhotos;
+  // };
```
Comment (mwazir) [inline src/App.js]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/App.js [hunk 12]
```diff
@@ -1,23 +1,132 @@
...
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
+      <header className="App-header wrapper">
+        <h1>City Information Guide</h1>
+        {/* <h2>Coming Soon!</h2> */}
```
Comment (mwazir) [inline src/App.js]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/App.js [hunk 13]
```diff
@@ -1,23 +1,132 @@
...
+
+        <CitySelectionForm getInfo={getInfo} />
+      </header>
+
+      {/* <DisplayInformation cityObject={cityInfo} /> */}
```
Comment (mwazir) [inline src/App.js]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/App.js [hunk 14]
```diff
@@ -1,23 +1,132 @@
...
+      <main>
+        {/* <h2>Select a city</h2> */}
```
Comment (mwazir) [inline src/App.js]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/components/CitySelectionForm.js [hunk 15]
```diff
@@ -0,0 +1,45 @@
+import "./../styles/CitySelectionForm.css";
+import { useState } from "react";
+
+function CitySelectionForm(props) {
+  const [userChoice, setUserChoice] = useState("placeholder");
+  // console.log("Inicial Users Choice State:", userChoice); //RETURNS placeholder  - OK
```
Comment (mwazir) [inline src/components/CitySelectionForm.js]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/components/CitySelectionForm.js [hunk 16]
```diff
@@ -0,0 +1,45 @@
...
+
+  //Get the User's choice value
+  const handleUserChoice = (e) => {
+    console.log("City from Form:", e.target.value);
```
Comment (mwazir) [inline src/components/CitySelectionForm.js]: console.log can be removed

File: src/components/CitySelectionForm.js [hunk 17]
```diff
@@ -0,0 +1,45 @@
...
+    //Update userChoice
+    setUserChoice(e.target.value);
+    // console.log("Users Choice State:", userChoice); //Why 1 behind???
```
Comment (mwazir) [inline src/components/CitySelectionForm.js]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/components/CitySelectionForm.js [hunk 18]
```diff
@@ -0,0 +1,45 @@
...
+  };
+
+  return (
+    <form onSubmit={(e) => props.getInfo(e, userChoice)} className="wrapper">
+      <label htmlFor="cityName">Choose city:</label>
+      {/* <label>Choose city:</label> */}
```
Comment (mwazir) [inline src/components/CitySelectionForm.js:16]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/components/DisplayPhoto.js [hunk 19]
```diff
@@ -0,0 +1,26 @@
...
+
+function DisplayPhotos(props) {
+  return (
+    <section>
+      {props.photos.length === 0 ? (
+        <h2>No photos found!</h2>
+      ) : (
+        <>
+          <h2>Photos</h2>
+          {/* {console.log("kkk", props.photos)} */}
```
Comment (mwazir) [inline src/components/DisplayPhoto.js]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/styles/App.css [hunk 20]
```diff
@@ -0,0 +1,52 @@
...
+background-color: #282c34;
+}
+
+.App {
+  text-align: center;  
+  color: beige;
+}
+
+header {
+  /* color: azure; */
```
Comment (mwazir) [inline src/styles/App.css]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/styles/App.css [hunk 21]
```diff
@@ -0,0 +1,52 @@
...
+  text-align: center;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+
+@media (min-width: 400px) {
+  header {
+    /* color: azure; */
+    /* height: 50vh; */
```
Comment (mwazir) [inline src/styles/App.css]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/styles/CitySelectionForm.css [hunk 22]
```diff
@@ -0,0 +1,31 @@
...
+    height: 40px;
+    margin: 20px auto;
+    border-radius: 5px;
+    text-align: center;
+}
+
+button {
+    width: 100%;
+    height: 40px;
+    /* margin-bottom: 25px; */
```
Comment (mwazir) [inline src/styles/CitySelectionForm.css]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/styles/App.css [hunk 23]
```diff
@@ -0,0 +1,52 @@
...
+  background-repeat: no-repeat;
+  background-size: contain; */
+}
+
+main p {
+  margin: 20px;
+}
+
+footer {
+  /* height: 10vh; */
```
Comment (mwazir) [inline src/styles/App.css]: Not sure if this is still being used, but if not then the commented code can be removed

File: src/styles/App.css [hunk 24]
```diff
@@ -0,0 +1,52 @@
...
+  height:50vh;
+}
+
+main {
+  /* height: 45vh; */
+  /* background-color: black; */
+  
+  /* background-image: url("./../assets/images/alberta-background.jpg");
+  background-repeat: no-repeat;
+  background-size: contain; */
```
Comment (mwazir) [inline src/styles/App.css]: Not sure if this is still being used, but if not then the commented code can be removed

**Review by mwazir — APPROVED** [review]
Hey Theo, finished your review. Let me know if you have any questions. Most of the comments are just commented code or console.logs that can be deleted

File: README.txt [hunk 25]
```diff
@@ -0,0 +1,47 @@
+1. PROJECT:
+
```
Comment (mwazir) [inline README.txt:2]: test

**Comment by tdmitchell** [comment]
Thanks for the notes. Already changed.

**Pull request closed without merging**